### ISSUE TITLE
Keyboard stays up after adding new search engine or autocomplete domain

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/focus-android#1923]
Not critical but a nice thing that a contributor could fix.

STR:
* Go to Settings -> Search -> Add another search engine
* Enter some random search engine URL and name.
* Press "Save"

Expected: Hide keyboard and return to the list of search engines.

Actual: Return to the list of search engines but keyboard is still up.

The same issue can be reproduced when adding a new autocomplete domain.

**Comment by Jacquester** [comment]
I'm not even able to add a search engine, as I select either one of the TextInputEditText it instantly loses focus.
Would you mind if I try my luck with this one ?

**Comment by pocmo** [comment]
> I'm not even able to add a search engine, as I select either one of the TextInputEditText it instantly loses focus.

@Jacquester This sounds like a severe bug. Can you file a separate issue for that and describe what you are doing. Please include device and Android version. Maybe even record a video if possible. Thank you!

>  Would you mind if I try my luck with this one ?

This issue is still available. Go for it!

**Comment by Jacquester** [comment]
> @Jacquester This sounds like a severe bug. Can you file a separate issue for that and describe what you are doing. Please include device and Android version. Maybe even record a video if possible. Thank you!

I'll get onto it as soon as possible :)

**Comment by liuche** [comment]
I wonder if this actually was a problem with adjustPan vs adjustResize in SettingsFragment - this is causing some conflicts with some patches we need to uplift to 4.0, so I'm going to temporarily back this out, and reland if this fix is still needed.

**Comment by liuche** [comment]
@Jacquester can you still repro this on a fresh pull from master?

**Comment by liuche** [comment]
Ah I can repro this on r26 devices (but not r23), will reland these changes.

**Comment by liuche** [comment]
Assigning myself to reland.

**Comment by liuche** [comment]
Actually (and sorry for so much back and forth) this is no longer an issue after #1936 .
